pkg/model/manifest.go

```diff
@@ -1360,3 +1360,19 @@ func (m *Manifest) GetStack() *Stack {
 	}
 	return m.Deploy.ComposeSection.Stack
 }
+
+func (m *Manifest) HasDependenciesSection() bool {
+	return m.IsV2 && len(m.Dependencies) > 0
+}
+
+func (m *Manifest) HasBuildSection() bool {
+	return m.IsV2 && len(m.Build) > 0
+}
+
+func (m *Manifest) HasDeploySection() bool {
```

Comment:
Do you think could we add some unit tests for this function? I've just noticed that the old implementation causes a panic in this block and it's affecting `2.16.x` at least.

It'd be good to make sure it's not happening again in the future

Reply:
I've added test for 3 helpers and updated the functions to contemplate the case of manifest being nil